rust/agama-server/src/software/web.rs

```diff
@@ -190,12 +194,23 @@ pub async fn software_service(dbus: zbus::Connection) -> Result<Router, ServiceE
     let software_issues = issues_router(&dbus, DBUS_SERVICE, DBUS_PATH).await?;
     let product_issues = issues_router(&dbus, DBUS_SERVICE, DBUS_PRODUCT_PATH).await?;
 
+    let mut licenses_repo = LicensesRepo::default();
```

Comment:
so if read below failed it will act as NulObject? And not display any license?

Reply:
It considers there repository is empty. We could add some logging.